pkg/database/realm.go

```diff
@@ -608,3 +608,27 @@ func (r *Realm) DestroySigningKeyVersion(ctx context.Context, db *Database, id i
 
 	return nil
 }
+
+// GetStats returns the usage statistics for this realm. If no stats exist,
+// returns an empty array.
+func (r *Realm) GetStats(db *Database, start, stop time.Time) ([]*RealmStats, error) {
```

Comment:
Per Go guidelines, I think this should just be `Stats()`?

Reply:
Done